docs/Release_notes.md

```diff
@@ -1,6 +1,199 @@
+-	PR 67: Added script to wait for CARLA. This script can replace sleep commands that were environment specific
+-	PR 68: Added workflow dispatch as carma-carla-integration tool depends on libraries in carla-sensor-lib. It should trigger rebuild if carla-sensorl-lib repo has new changes pushed.
+
+Fixes in this release:
+-	PR 64: Fixed error log statements caused by incorrect syntax in launch file.
+
+### **CARMA NS3 Adapter** 
+
+Enhancement in the release:
+-	PR 11: Added udp listener to listen to time sync messages from CDASim and broadcast it in rostopic as /sim_clock.
+-	PR 13: Added wave JSON configurations for SDSM.
+
+### **CARLA Sensor Library** 
+
+The carla-sensor-lib is a new repository and docker image for this release. This library provides functions to create CARLA sensors with noise modeling and retrieve detection information from created sensors. Additionally this repository creates a docker image which deploys and XML RPC server that allows clients to create sensor in CARLA and poll their detections. This new feature is provided to enable object detection for CDASim deployment using CARLA's existing sensor and object representation. The data will be feed to the CARMA-Platform via the CARLA CARMA Integration tool, and to CDASim to expose the data to other integrated simulators or software systems under test.
+
+### **CARLA ScenarioRunner** 
+
+The scenario-runner is a new repository and docker image for this release. This directory contains custom ScenarioRunner scenario configurations to facilitate integration testing. This still a work in progress, so the scenarios serve more as example references. ScenarioRunner's code resides in the srunner Python package, and the scenario_runner.py script is responsible for launching scenarios. 
+
+### **EVC-SUMO (New Private Repository)** 
+
+This EVC-sumo is a new private repository. Econolite Virtual Controller (EVC)-SUMO bridge tool that is a part of XiL co-simulation tool. This bridge acts as a mediator between EVC (via the PyEOS Python package) and SUMO, integrating the exchange of traffic light status and detector status.
+Currently, in order to build and use the docker image, the tool needs proprietary github-token accessible to the members of the usdot-fhwa-stol organization due to Econolite's license.
+
+### **Multiple Object Tracking** 
+
+Multiple Object Tracking is a new repository that holds a library to support cooperative perception. This library is intended for tracking multiple objects from multiple sources where the input is abstracted as a standard cooperative perception object tracking interface. For example, object level data can come from multiple sources including J2334 Sensor Data Sharing Message, Basic Safety Message, or local perception. This enables the library to be deployed in different road actors such as C-ADS equipped vehicles or the infrastructure with the help of message adapters that are suited for respective middleware. 
```

Comment:
The Multiple Object Tracking repository introduces a new library designed to support cooperative perception. This library is tailored for tracking multiple objects from various sources, utilizing a standard cooperative perception object tracking interface. Inputs can include object-level data from diverse sources such as the J2334 Sensor Data Sharing Message, Basic Safety Message, or local perception. This versatility allows the library to be deployed across different road actors, such as C-ADS equipped vehicles or infrastructure, using message adapters appropriate for the respective middleware.

The library features multiple submodules and functionalities that leverage architectural and algorithmic advancements from the sensor fusion community. An example of how this library can be implemented to execute a complete multiple object tracking pipeline is demonstrated in the CARMA Platform.

Reply:
updated